content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2013,18 +2060,31 @@ The name of the `fields` argument on the `@@unique` attribute can be omitted:
 @@unique(fields: [title, author], name: "titleAuthor")
 ```
 
+**Note:** In version 3.5.0 and later, new `length` and `sort` arguments are available as a Preview feature. See [Index configuration](../../../../concepts/components/prisma-schema/index-configuration) for more details.
+
+| Name     | Required | Type     | Description                                                                                            |
+| -------- | -------- | -------- | ------------------------------------------------------------------------------------------------------ |
+| `length` | **No**   | `number` | **MySQL only.** Allows you to specify a maximum length for the subpart of the value to be indexed.     |
+| `sort`   | **No**   | `String` | Allows you to specify in what order the entries of the index or constraint are stored in the database. |
+
 #### Signature
 
 ```prisma no-lines
 @@unique(_ fields: FieldReference[], name: String?, map: String?)
 ```
 
-> **Note**: Until version 3.0.0, the signiture was:
+> **Note**: Until version 3.0.0, the signature was:
 >
 > ```prisma no-lines
 > @@unique(_ fields: FieldReference[], name: String?)
 > ```
 
+> **Note**: In version 3.5.0 and later, the signature for the `"extendedIndexes"` Preview feature is:
+>
+> ```prisma no-lines
```

Comment:
Same as for @@id

Reply:
Also this contains a copypasted @id I think.

Reply:
Oops, thanks for catching the copypastes